cranelift-codegen/meta/src/cdsl/instructions.rs

```diff
@@ -527,57 +525,51 @@ fn verify_polymorphic(
 ///
 /// All polymorphic results must be derived from `ctrl_typevar`.
 ///
-/// Return a vector of other type variables used, or panics.
-fn verify_ctrl_typevar(
+/// Return a vector of other type variables used, or a string explaining what went wrong.
+fn is_ctrl_typevar_candidate(
```

Comment:
We might consider tests for `is_ctrl_typevar_candidate` and `verify_polymorphic`; not because they are so very complex but a couple of illustrative tests might explain how these functions work and ensure that when I touch this file I don't mess up the intended behavior 😄.